test_setup.js

```diff
@@ -148,6 +159,34 @@ async function setupGoogleProjectDynamic() {
     }
   }
 
+  // clear old keys from the service accounts in the dcf-integration GCP project
+  const dcfSvcAccounts = await google.listServiceAccounts('dcf-integration');
+  // console.log('#### ##: ' + dcfSvcAccounts);
```

Comment:
Run this as a `before` hook for whatever test suite tries to create new keys

Reply:
This whole logic runs within a `setupGoogleProjectDynamic` function, that is only called when a `@reqGoogle` tag is included in the overall test run. You can even find some old service-account-related logic inside the same block.
```
    if (isIncluded('@reqGoogle')) {
      createGoogleTestBuckets();
```

I don't know what was the rationale back then.
Do you want me to create a separate ticket / PR to address this tech debt in a more comprehensive way? I was hoping to merge this change to avoid more intermittent failures.

Reply:
Moving this to `googleServiceAccountKeyTest.js`.